src/sqlite3.cr

```diff
@@ -2,7 +2,8 @@ require "db"
 require "./sqlite3/**"
 
 module SQLite3
-  DATE_FORMAT = "%F %H:%M:%S.%L"
+  DATE_FORMAT        = "%F %H:%M:%S.%L"
+  DATE_FORMAT_NO_DOT = "%F %H:%M:%S"
```

Comment:
Can we have a more descriptive differentiation than `NOT_DOT`? One is with sub-second precision, the other without. So maybe `DATE_FORMAT_SECOND` and `DATE_FORMAT_SUBSECOND`?